src/Cassandra/Mapping/Statements/CqlGenerator.cs

```diff
@@ -45,10 +45,11 @@ public void AddSelect<T>(Cql cql)
             var pocoData = _pocoDataFactory.GetPocoData<T>();
             var allColumns = pocoData.Columns.Select(Escape(pocoData)).ToCommaDelimitedString();
 
+            var identifier = string.IsNullOrEmpty(pocoData.KeyspaceName) ? pocoData.TableName : string.Format("{0}.{1}", pocoData.KeyspaceName, pocoData.TableName);
```

Comment:
The fully-qualified escaped table name should be of the form: `"Ks1"."MyTable"` (not `"Ks1.MyTable"`)